src/control-plane/backend/lambda/api/common/utils.ts

```diff
@@ -435,6 +435,12 @@ function _checkInterfaceEndpoint(allSubnets: ClickStreamSubnet[], isolatedSubnet
       reason: 'The traffic is not allowed by security group rules',
     });
   }
+  if (service.includes('glue') && !_isAllowAllIngressTrafficFromSubnets(vpcEndpointSGRules, subnet)) {
```

Comment:
Why only check for the `glue` endpoint? We need to validate the security groups of all VPC endpoints used by the data pipeline.

Reply:
All endpoints will check ingress from isolated subnets.